webknossos/webknossos/dataset/downsampling_utils.py

```diff
@@ -282,18 +278,10 @@ def downsample_cube_job(
     mag_factors: List[int],
     interpolation_mode: InterpolationModes,
     buffer_edge_len: int,
-    job_count_per_log: int,
 ) -> None:
-    (source_view, target_view, i) = args
-    use_logging = i % job_count_per_log == 0
-
-    if use_logging:
-        logging.info(f"Downsampling of {target_view.global_offset}")
+    (source_view, target_view, _i) = args
```

Comment:
The `i` is added by the functions `for_each_chunk` and `for_zipped_chunks` automatically. The initial purpose of this parameter was to be used for logging, which is now no longer needed. However, removing it now would require that each "worker"-method is updated. This would then be a breaking change. Therefore, I think it is fine if we leave it like this for now.